index.bs

```diff
@@ -2108,29 +2113,29 @@ Sensitive Information {#sensitive-information-header}
 
-### Trustworthy documents and origins ### {#trustworthy-documents}
-In order to expose any [=sensitive information=] the requesting document MUST be considered [=trustworthy=].
+### Active and focused document ### {#active-and-focused-document}
+A document MUST be [=active and focused=] at the time that [=sensitive information=] is requested.
 
-<div class="algorithm" data-algorithm="document-is-trustworthy">
+<div class="algorithm" data-algorithm="document-is-active-and-focused">
 
-To determine if a given {{Document}} |document| is <dfn>trustworthy</dfn> the user agent MUST run the following steps:
+To determine if a given {{Document}} |document| is <dfn>active and focused</dfn> the user agent MUST run the following steps:
 
-  1. If |document| is not a [=responsible=] document, return <code>false</code>
-  1. If |document| is not of a [=secure context=], return <code>false</code>
-  1. If |document|'s origin is blocked by [[#feature-policy|feature policy]], return <code>false</code>
+  1. If the [=currently focused area=] does not belong to |document|, return <code>false</code>
+  1. If |document| is not of the [=same origin-domain=] as the [=active document=], return <code>false</code>
   1. Return <code>true</code>
 
 </div>
 
-### Active and focused document ### {#active-and-focused-document}
-In addition to being [=trustworthy=], a document MUST be [=active and focused=] at the time that [=sensitive information=] is requested.
+### Trustworthy documents and origins ### {#trustworthy-documents}
+In order to expose any [=sensitive information=] the requesting document MUST be considered [=trustworthy=].
 
-<div class="algorithm" data-algorithm="document-is-active-and-focused">
+<div class="algorithm" data-algorithm="document-is-trustworthy">
 
-To determine if a given {{Document}} |document| is <dfn>active and focused</dfn> the user agent MUST run the following steps:
+To determine if a given {{Document}} |document| is <dfn>trustworthy</dfn> the user agent MUST run the following steps:
 
-  1. If the [=currently focused area=] does not belong to |document|, return <code>false</code>
-  1. If |document| is not of the [=same origin-domain=] as the [=active document=], return <code>false</code>
+  1. If |document| is not a [=responsible=] document, return <code>false</code>
+  1. If |document| is not [=active and focused=], return <code>false</code>
+  1. If |document|'s origin is not allowed to use the WebXR [[#feature-policy|feature policy]] or the [=underlying sensors=]' feature policies, return <code>false</code>
```

Comment:
@johnpallett Is there already an issue tracking the need to make a decision on this topic?

Reply:
Good question; #768 contains the considerations and raises the question. It could be used either to track a decision about whether we should remove the requirement in the short-term, or could be used to find a longer-term solution. @ddorwin @NellWaliczek do you have a preference for how that conversation is structured?

Reply:
I'm going to accept John's suggestion here because it seems like the overall lowest impact way to proceed forward for the following reason:

The intent behind this was to prevent pages from "backdooring" certain bits of accelerometer/gyro sensor data that may reveal things like pin entry on a 2D page. I think it's reasonable to say that this should not be a concern for `immersive-vr` because the context in which the sensors are being used is significantly different and has a different set of threat vectors (the handling of which is what this entire section is about.) The biggest concerning scenario, as such, is sensor-based tracking in `inline` mode. Fortunately for us this is not something with guaranteed availability. It may be rejected by the user or the UA, or simply not be supported by the device at all (desktop.) As such, if we were to introduce a similar restriction down the line the actual impact of it would be that some mobile devices start displaying the same inline behavior as a desktop device. That seems like a pretty soft failure, as these things go. Plus, it gives a clear indication that something has stopped working and gives us an opportunity to communicate why in the console at that point. If we went the other way and decided that these additional feature policies were not, in fact, necessary we would have a much more difficult time communicating to developers that they have now exposed certain features unnecessarily, since nothing would indicate that there was a problem to look into and it would be a mighty heuristics challenge to identify scenarios in which the developer ONLY was using those feature policies for WebXR and as such we could safely tell them to remove them.